packages/paste-core/components/switch/src/types.ts

```diff
@@ -0,0 +1,35 @@
+import type React from 'react';
+import type {BoxProps} from '@twilio-paste/box';
+import type {TextProps} from '@twilio-paste/text';
+
+export interface SwitchContainerProps extends React.HTMLAttributes<HTMLLabelElement>, Pick<BoxProps, 'element'> {
+  children: NonNullable<React.ReactNode>;
+  disabled?: boolean;
+  helpText?: string | React.ReactNode;
+  label?: string | React.ReactNode;
+  required?: boolean;
+}
+
+export interface SwitchProps extends React.HTMLAttributes<HTMLDivElement>, Pick<BoxProps, 'element'> {
+  id?: string;
+  disabled?: boolean;
+  on?: boolean;
+  labelId?: string;
```

Comment:
question (non-blocking): what's the difference in use-case between labelId and id? Can this work with just one id?

Reply:
using a labelId allows us to use the required aria-labelledby prop